client/platform/desktop/frontend/components/Export.vue

```diff
@@ -0,0 +1,151 @@
+      type: String,
+      required: true,
+    },
+    small: {
+      type: Boolean,
+      default: false,
+    },
+  },
+
+  setup(props) {
+    const data = reactive({
+      menuOpen: false,
+      excludeFiltered: false,
+      activator: 0,
+      err: null as unknown,
+      meta: null as JsonMeta | null,
+      outPath: '',
+    });
+
+    watch(toRef(data, 'menuOpen'), async (newval) => {
+      if (newval) {
+        data.meta = await loadMetadata(props.id);
+      } else {
+        data.err = null;
+        data.outPath = '';
+      }
+    });
+
+    const thresholds = computed(() => (
+      data.meta
+        ? Object.keys(data.meta.confidenceFilters || {})
+        : []));
+
+    async function doExport() {
+      try {
+        data.err = null;
+        data.outPath = await exportDataset(props.id, data.excludeFiltered);
+      } catch (err) {
+        data.err = err;
+        throw err;
+      }
+    }
+
+    return {
+      data,
+      doExport,
+      thresholds,
+    };
+  },
+});
+</script>
+
+<template>
+  <v-menu
+    v-model="data.menuOpen"
+    :close-on-content-click="false"
+    :nudge-width="280"
+    offset-y
+    max-width="280"
+  >
+    <template #activator="{ on: menuOn }">
+      <v-tooltip bottom>
+        <template #activator="{ on: tooltipOn }">
+          <v-btn
+            class="ma-0"
+            text
+            :small="small"
+            v-on="{ ...tooltipOn, ...menuOn }"
+          >
+            <v-icon color="accent">
+              mdi-export
+            </v-icon>
+            <span
+              v-show="!$vuetify.breakpoint.mdAndDown"
+              class="pl-1"
+            >
+              Export
+            </span>
+          </v-btn>
+        </template>
+        <span>export annotation data</span>
+      </v-tooltip>
+    </template>
+    <template>
+      <v-card v-if="data.menuOpen">
+        <v-card-title>
+          Export options
+        </v-card-title>
+
+        <v-card-text class="pb-0">
+          <v-alert
+            v-if="data.err"
+            type="error"
+          >
+            {{ data.err }}
+          </v-alert>
+          <v-alert
+            v-if="data.outPath"
+            dense
+            class="text-caption"
+            type="success"
+          >
+            Export succeeded.
+          </v-alert>
+          <div>Export to VIAME CSV format</div>
+          <template v-if="thresholds.length">
+            <v-checkbox
+              v-model="data.excludeFiltered"
+              label="exclude tracks below confidence threshold"
+              dense
+              hide-details
+            />
+            <div
+              v-if="data.meta && data.meta.confidenceFilters"
+              class="py-2"
+            >
+              <div>Current thresholds:</div>
+              <span
+                v-for="(val, key) in data.meta.confidenceFilters"
+                :key="key"
+                class="pt-2"
+              >
+                ({{ key }}, {{ val }})
```

Comment:
I didn't like the idea of having more than one source of truth for that value, and importing it from `useTrackFilters` would be inconsistent since the default is established at runtime.

Should do something about this, but none of the options seemed appropriate, so I ignored it.